yarn.lock

```diff
@@ -2394,11 +2394,6 @@ babel-plugin-module-resolver@^4.0.0:
     reselect "^4.0.0"
     resolve "^1.13.1"
 
-babel-plugin-syntax-dynamic-import@^6.18.0:
-  version "6.18.0"
-  resolved "https://registry.yarnpkg.com/babel-plugin-syntax-dynamic-import/-/babel-plugin-syntax-dynamic-import-6.18.0.tgz#8d6a26229c83745a9982a441051572caa179b1da"
-  integrity sha1-jWomIpyDdFqZgqRBBRVyyqF5sdo=
-
```

Comment:
You'll notice quickly that not much is actually removed from the yarn.lock. This is because they're transitive dependencies.
`babel-runtime` should disappear once I manage to upgrade flow-coverage-report, but `@babel/runtime` is still pulled by some dependencies. Hopefully this doesn't get in our bundle in the end...